packages/vscode-extension/package.nls.json

```diff
@@ -400,9 +400,8 @@
     "teamstoolkit.taskDefinitions.args.forceUpdate.false.description": "false: only install when dependency changes.",
     "teamstoolkit.taskDefinitions.args.forceUpdate.true.description": "true: always run npm install script.",
     "teamstoolkit.taskDefinitions.args.configFile.title": "The configuration file of the ngrok tunnels.",
-    "teamstoolkit.taskDefinitions.args.binFolder.title":"The ngrok bin folder.",
+    "teamstoolkit.taskDefinitions.args.useGlobalNgrok.title":"Use the ngrok installed by Teams Toolkit or use the global ngrok.",
```

Comment:
If true, use global ngrok from PATH instead of the Teams Toolkit built-in one.

Reply:
Updated.